### DISCUSSION TITLE
https://blog-umber-sigma.vercel.app/%E7%AC%94%E8%AE%B0/%E6%BC%94%E7%A4%BA%E7%9B%AE%E5%BD%951/%E6%96%87%E6%A1%A3-%E5%B8%A6%E8%AF%84%E8%AE%BA.html

### DISCUSSION THREAD

**Discussion by giscus** [Jackiexiao/nolebase-template#5] [Announcements]
# https://blog-umber-sigma.vercel.app/%E7%AC%94%E8%AE%B0/%E6%BC%94%E7%A4%BA%E7%9B%AE%E5%BD%951/%E6%96%87%E6%A1%A3-%E5%B8%A6%E8%AF%84%E8%AE%BA.html

记录回忆，知识和畅想的地方

https://blog-umber-sigma.vercel.app/%E7%AC%94%E8%AE%B0/%E6%BC%94%E7%A4%BA%E7%9B%AE%E5%BD%951/%E6%96%87%E6%A1%A3-%E5%B8%A6%E8%AF%84%E8%AE%BA.html

**Comment by hyoukann** [comment]
请问企鹅
